### DISCUSSION TITLE
How to zoom in on a section under a heading level?

### DISCUSSION THREAD

**Discussion by rsomani95** [vslinko/obsidian-zoom#19] [Q&A] [answered]
Hi! Thanks for your work on this and the outliner plugin!

I was curious if it would be possible to zoom in on a section under a heading level.
```markdown
## Section 1
Dummy text

### Sub Section 1
More dummy text

### Sub Section 2
Some more dummy text
```

In this particular example, it may make more sense to write it out as a list, but imagine longer documents with paragraphs. One could argue that the right approach there would be to split the document into multiple notes, but I'm curious:
1. If the above mentioned feature is possible to implement?
2. Do you think it's worth implementing?

Thanks!

**Comment by vslinko** [comment, marked as answer]
Hi @rsomani95 !

If I understand you correctly, this is already implemented.

Please try to put the cursor on the heading and press <kbd>Cmd</kbd><kbd>.</kbd> or <kbd>Ctrl</kbd><kbd>.</kbd>.

**Reply by rsomani95** [reply]
Thanks @vslinko, will do!
